pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[project]
+name = "dicom_anonymizer"
+version = "1.0.11"
+authors = [
+  { name="Laurenn Lam", email="laurenn.lam@kitware.com" },
+]
+description = "Program to anonymize dicom files with default and custom rules"
+readme = "README.md"
+requires-python = ">=3.10"

```

Comment:
Why restrict to 3.10 and higher ? I tested with Python 3.9 and 3.8 and it seems to work

Reply:
That was my confusion. Per [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata) documentation, it was "provisional" until 3.10, so I thought it was not available for versions 3.8 and 3.9. Since these worked, I will update this minimum requirement to 3.8.

Thank you for testing!

Reply:
Fixed in commit b3dc158

Reply:
Thank you! I will merge the main branch into my branch and then clean up my PR.